Exercise1/Adapter/AndroidInfoAdapter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using Android.Graphics;
+using Android.Support.V7.Widget;
+using Android.Util;
+using Android.Views;
+using Exercise1.Models;
+
+namespace Exercise1.Adapter
+{
+    class AndroidInfoAdapter : RecyclerView.Adapter
+    {
+        private List<VersionInfo> versionInfos;
+
+        public AndroidInfoAdapter(List<VersionInfo> versionInfos)
+        {
+            this.versionInfos = versionInfos;
+        }
+
+        public override void OnBindViewHolder(RecyclerView.ViewHolder holder, int position)
+        {
+            if (!(holder is AndroidViewHolder vh)) return;
```

Comment:
You should create a property `VersionInfo ` into the class AndroidViewHolder
```C#
class AndroidViewHolder...{
.....
    public VersionInfo VersionInfo{
        set{
             TvName.Text = value.Name;
             TvVersion.Text = value.Version;
             Img.SetImageBitmap(Base64ToBitmap(value.Image));
     }
.....
}
```
So, here, the code should be clean
```C#
if (holder is AndroidViewHolder vh){
     vh.VersionInfo = versionInfos[position];
}
```

You also can remove `if (holder is AndroidViewHolder vh){` here because it isn't very necessary